soh/Makefile

```diff
@@ -53,27 +73,60 @@ INC_DIRS := $(addprefix -I, \
 	../libultraship/libultraship/Lib/Fast3D/U64/PR \
 )
 
+ifeq ($(UNAME), Linux) #LINUX
+	INC_DIRS += $(addprefix -I, \
+		/opt/X11/include \
+)
+endif
+
+ifeq ($(UNAME), Darwin) #APPLE
+	INC_DIRS += $(shell pkg-config --cflags x11)
+endif
+
 LDDIRS := $(addprefix -L, \
 	../external \
 	../libultraship/ \
 )
 
+ifeq ($(UNAME), Linux) #LINUX
+	LDDIRS += $(addprefix -L, \
+		/opt/X11/lib \
+)
+endif
+
+ifeq ($(UNAME), Darwin) #APPLE
+	LDDIRS += $(shell pkg-config --libs x11)
+endif
+
 LDLIBS := \
 	$(ZAPDUTILS) \
 	$(addprefix -l, \
-	X11 \
 	dl \
 	bz2 \
 	z \
 	pthread \
 	atomic \
-	SDL2 \
```

Comment:
SDL2 and GLEW were removed from LDLIBS from linux. 
Darwin uses `$(shell sdl2-config --libs) $(shell pkg-config --libs glew)`
Linux should have similar or add SDL2/GLEW back
```
ifeq ($(UNAME), Linux) #LINUX
LDLIBS += \
```

Reply:
Ah yes! It should be added below in the Linux only section